### ISSUE TITLE
Tubitv.com downloads are broken

### ISSUE THREAD

**Issue by tjamqt** [ytdl-org/youtube-dl#27140]
## Checklist

- [x ] I'm reporting a broken site support
- [x ] I've verified that I'm running youtube-dl version **2020.11.21.1**
- [ ] I've checked that all provided URLs are alive and playable in a browser
- [x ] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x ] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
D:\Video\Movies>C:\Util\ffmpeg\bin\youtube-dl -v "https://tubitv.com/movies/556756/chappie"
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://tubitv.com/movies/556756/chappie']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2020.11.21.1
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.16299
[debug] exe versions: ffmpeg N-94391-g052d41377a, ffprobe N-94391-g052d41377a
[debug] Proxy map: {}
[debug] Using fake IP 6.164.26.159 (US) as X-Forwarded-For.
[TubiTv] 556756: Downloading JSON metadata
[TubiTv] 556756: Downloading m3u8 information
Traceback (most recent call last):
  File "__main__.py", line 19, in <module>
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\__init__.py", line 474, in main
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\__init__.py", line 464, in _real_main
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\YoutubeDL.py", line 2029, in download
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\YoutubeDL.py", line 796, in extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\YoutubeDL.py", line 803, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\YoutubeDL.py", line 824, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\extractor\common.py", line 532, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\extractor\tubitv.py", line 67, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\extractor\common.py", line 1628, in _extract_m3u8_formats
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\extractor\common.py", line 665, in _download_webpage_handle
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\extractor\common.py", line 627, in _request_webpage
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmp6zrp041h\build\youtube_dl\utils.py", line 2157, in sanitized_Request
  File "C:\Python\Python34\lib\urllib\request.py", line 267, in __init__
  File "C:\Python\Python34\lib\urllib\request.py", line 293, in full_url
  File "C:\Python\Python34\lib\urllib\request.py", line 322, in _parse
ValueError: unknown url type: ''

```

## Description

Maybe half of the URLs on the tubitv.com website can be downloaded without incident but this one and the rest result in the above error.

I also downloaded Python source code from github and the extractor test fails with some similarity in the stack trace (Anaconda, Python 3.6.5) :

(base) C:\SoftDev\Python\youtube-dl-master> python test/test_download.py TestDownload.test_TubiTv
[debug] Using fake IP 6.208.88.246 (US) as X-Forwarded-For.
[TubiTv] 283829: Downloading JSON metadata
[TubiTv] 283829: Downloading m3u8 information
E
======================================================================
ERROR: test_TubiTv (__main__.TestDownload):
----------------------------------------------------------------------
Traceback (most recent call last):
  File "test/test_download.py", line 159, in test_template
    force_generic_extractor=params.get('force_generic_extractor', False))
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\YoutubeDL.py", line 796, in extract_info
    return self.__extract_info(url, ie, download, extra_info, process)
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\YoutubeDL.py", line 803, in wrapper
    return func(self, *args, **kwargs)
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\YoutubeDL.py", line 824, in __extract_info
    ie_result = ie.extract(url)
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\extractor\common.py", line 532, in extract
    ie_result = self._real_extract(url)
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\extractor\tubitv.py", line 67, in _real_extract
    video_id, 'mp4', 'm3u8_native')
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\extractor\common.py", line 1628, in _extract_m3u8_formats
    fatal=fatal, data=data, headers=headers, query=query)
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\extractor\common.py", line 665, in _download_webpage_handle
    urlh = self._request_webpage(url_or_request, video_id, note, errnote, fatal, data=data, headers=headers, query=query, expected_status=expected_status)
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\extractor\common.py", line 627, in _request_webpage
    url_or_request = sanitized_Request(url_or_request, data, headers)
  File "C:\SoftDev\Python\youtube-dl-master\youtube_dl\utils.py", line 2157, in sanitized_Request
    return compat_urllib_request.Request(sanitize_url(url), *args, **kwargs)
  File "C:\ProgramData\Anaconda3\lib\urllib\request.py", line 329, in __init__
    self.full_url = url
  File "C:\ProgramData\Anaconda3\lib\urllib\request.py", line 355, in full_url
    self._parse()
  File "C:\ProgramData\Anaconda3\lib\urllib\request.py", line 384, in _parse
    raise ValueError("unknown url type: %r" % self.full_url)
ValueError: unknown url type: ''

----------------------------------------------------------------------
Ran 1 test in 1.242s

FAILED (errors=1)

**Comment by remitamine** [comment]
youtube-dl does not support DRM protected videos.

**Comment by tjamqt** [comment]
OK, thanks for checking!

    On Saturday, November 21, 2020, 5:01:20 PM EST, remitamine <notifications@github.com> wrote:  
 
 


Closed #27140.
